src/its_preselector/preselector.py

```diff
@@ -77,62 +79,114 @@ def __set_cal_sources(self, cal_sources):
-        return None
-
-    def get_frequency_high_passband(self, rf_path_index):
-        if rf_path_index < len(self.rf_paths):
-            path = self.rf_paths[rf_path_index]
+            preselector_filter = self.__get_filter(filter_id)
+            if preselector_filter:
+                return preselector_filter.frequency_low_passband
+            else:
+                raise ConfigurationException(
+                    "Unable to get frequency_low for the passband filter. There is no RF_PATH named {path_name}".format(
+                        path_name=rf_path_name))
+
+    def get_frequency_high_passband(self, rf_path_name: str) -> float:
+        """
+        Get the high frequency of the 1 dB passband.
+        :param rf_path_name: The name of the rf_path.
+        :return: The high frequency of the 1 dB passband in Hz.
+        """
+        if rf_path_name in self.rf_paths:
+            path = self.rf_paths[rf_path_name]
             filter_id = path.filter_id
             preselector_filter = self.__get_filter(filter_id)
             if preselector_filter:
                 return preselector_filter.frequency_high_passband
-        return None
-
-    def get_frequency_low_stopband(self, rf_path_index):
-        if rf_path_index < len(self.rf_paths):
-            path = self.rf_paths[rf_path_index]
+        else:
+            raise ConfigurationException(
+                "Unable to get frequency_high for the passband filter. There is no RF_PATH named {path_name}".format(
+                    path_name=rf_path_name))
+
+    def get_frequency_low_stopband(self, rf_path_name: str):
+        """
+        Gets the low frequency of the 60 dB stopband.
+        :param rf_path_name: the name of the rf_path.
+        :return: The low frequency of the 60 dB stopband in Hz.
+        """
+        if rf_path_name in self.rf_paths:
+            path = self.rf_paths[rf_path_name]
             filter_id = path.filter_id
             preselector_filter = self.__get_filter(filter_id)
             if preselector_filter:
                 return preselector_filter.frequency_low_stopband
-        return None
-
-    def get_frequency_high_stopband(self, rf_path_index):
-        if rf_path_index < len(self.rf_paths):
-            path = self.rf_paths[rf_path_index]
+        else:
+            raise ConfigurationException(
+                "Unable to get frequency_low for the stopband filter. There is no RF_PATH named {path_name}".format(
+                    path_name=rf_path_name))
+
+    def get_frequency_high_stopband(self, rf_path_name: str) -> float:
+        """
+        Get the high frequency of the 60 dB stopband.
+        :param rf_path_name:
+        :return: The high frequency of the 60 dB stopband in Hz.
+        """
+        if rf_path_name in self.rf_paths:
+            path = self.rf_paths[rf_path_name]
             filter_id = path.filter_id
             preselctor_filter = self.__get_filter(filter_id)
```

Comment:
Typo in `preselctor_filter` here and in the following couple of lines

Reply:
Fixed in [f3d8e3f](https://github.com/NTIA/Preselector/pull/11/commits/f3d8e3f807943b060f9eec493b9e62351366b5fb)